docs/_redirects

```diff
@@ -16,4 +16,5 @@
 /tools/vscode/en/lwc                            /tools/vscode/en/lwc/writing            302
 /tools/vscode/jp/lwc                            /tools/vscode/jp/lwc/writing            302
 /tools/vscode/en/visualforce                    /tools/vscode/en/visualforce/writing    302
-/tools/vscode/jp/visualforce                    /tools/vscode/jp/visualforce/writing    302
\ No newline at end of file
+/tools/vscode/jp/visualforce                    /tools/vscode/jp/visualforce/writing    302
+/tools/vscode/en/lwc/write-lwc                  /tools/vscode/en/lwc/writing            302
```

Comment:
This PR changes the links in the files from `lwc/write-lwc` to `lwc/writing` do we need to keep this last redirect ?

Reply:
The page was already public for a week or two so this is just making sure anyone who used it or if google indexed it doesnt get a 404.